app/models/podcast_import.rb

```diff
@@ -67,12 +67,14 @@ def validate_feed(podcast_feed)
   end
 
   def create_series_from_podcast(feed = self.feed)
+    description = [feed.description, feed.itunes_summary].max if feed.itunes_summary
+
     # create the series
     self.series = create_series!(
       account: account,
       title: feed.title,
       short_description: feed.itunes_subtitle,
-      description: feed.description
+      description: description || feed.description
```

Comment:
seems odd to set description based on `feed.description`, but then to also have it in this expression.